### PR TITLE
feat: add summerfi aaveV3 import recipe

### PR DESCRIPTION
**Pull request by kjaredic**

### CODE REVIEW

**Review by rajko-z — APPROVED** [review]

File: contracts/actions/summerfi/SFApproveTokens.sol [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+import {ActionBase} from '../ActionBase.sol';
+import {IERC20} from '../../interfaces/IERC20.sol';
+import {IDSProxy} from '../../interfaces/IDSProxy.sol';
+import {IExecutable} from '../../interfaces/summerfi/IExecutable.sol';
+import {IOperationsRegistry} from '../../interfaces/summerfi/IOperationsRegistry.sol';
+import {IOperationExecutor, Call} from '../../interfaces/summerfi/IOperationExecutor.sol';
+
+
+/// @title Approve tokens through Summerfi proxy
+contract SFApproveTokens is ActionBase {
```
Comment (rajko-z) [inline contracts/actions/summerfi/SFApproveTokens.sol:13]: Would add comment that we need to permit address that will call through Summerfi proxy before calling this action

File: contracts/actions/summerfi/SFApproveTokens.sol [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    address constant SF_OPERATION_EXECUTOR = 0xcA71C36D26f515AD0cce1D806B231CBC1185CdfC;
+    address constant SF_OPERATIONS_REGISTRY = 0x563d2689fE89c78259dD7F694146BB93f6388A55;
+    string constant SF_OPERATION_NAME = 'AAVEV3PaybackWithdraw';
```
Comment (rajko-z) [inline contracts/actions/summerfi/SFApproveTokens.sol:16]: We usually use double quotes in contracts, same goes for imports 😄

File: test/recipes/summerfi-shifter/summerfi-aaveV3-import.js [hunk 3]
```diff
@@ -0,0 +1,399 @@
...
+
+describe('Summerfi-AaveV3-Import', function () {
+    this.timeout(1_000_000);
+    let flAddress;
+
+    const fixture = async () => {
+        await resetForkToBlock(sfProxyLiquidBlock);
+
+        await redeploy('SFApproveTokens');
+        await getContractFromRegistry('AaveV3View');
```
Comment (rajko-z) [inline test/recipes/summerfi-shifter/summerfi-aaveV3-import.js:300]: Not used

Comment (kjaredic) [reply test/recipes/summerfi-shifter/summerfi-aaveV3-import.js:300]: It conditionally redeploys if not in registry. AaveV3View is used later on.

File: contracts/actions/summerfi/SFApproveTokens.sol (hunk 2 above)
Comment (kjaredic) [reply contracts/actions/summerfi/SFApproveTokens.sol:16]: auto format, my bad :(

File: contracts/actions/summerfi/SFApproveTokens.sol (hunk 1 above)
Comment (kjaredic) [reply contracts/actions/summerfi/SFApproveTokens.sol:13]: legit

**Review by rajko-z — APPROVED** [review]

**Pull request closed without merging**